examples/Bunch/include/simulation_defines/param/fieldBackground.param

```diff
@@ -0,0 +1,222 @@
+/**
+ * Copyright 2014-2015 Axel Huebl, Alexander Debus
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+/** Load pre-defined templates */
+#include "fields/background/templates/bgrTWTS.def"
+
+#ifndef PARAM_INCLUDE_FIELDBACKGROUND
+#define PARAM_INCLUDE_FIELDBACKGROUND false
+#endif
+
+/** Load external background fields
+ *
+ */
+namespace picongpu
+{
+    class FieldBackgroundE
+    {
+    public:
+    
+        /* Add this additional field for pushing particles */
+        static const bool InfluenceParticlePusher = PARAM_INCLUDE_FIELDBACKGROUND;
+
+        /* We use this to calculate your SI input back to our unit system */
+        const float3_64 unitField;
+
+        /* TWTS E-fields need to be initialized on host,
+         *  so they can look up global grid dimensions. */
+        const templates::TWTSFieldE twtsFieldE;
+        
+        /* Constructor is host-only, because of subGrid and halfSimSize initialization */
+        HINLINE FieldBackgroundE( const float3_64 unitField ) : unitField(unitField),
+        
+            twtsFieldE( 30.0e-6,  /* focus_y [m], the distance to the laser focus in y-direction */
+                /* wavelength [m] */
+                0.8e-6,
+                /* pulselength [s], sigma of std. gauss for intensity (E^2) */
+                10.0e-15 / 2.3548200450309493820231386529194,
+                /* w_x [m], cylindrically focused spot size */
+                5.0e-6,
+                /* w_y [m] */
+                0.01,
+                /* interaction angle between TWTS laser propagation vector and the y-axis [rad] */
+                60. *(PI/180.),
+                /* propagation speed of overlap [speed of light]. */
+                1.0,
+                /* manual time delay [s] if auto_tdelay is false */
+                ( 39.3e-6 )/(::picongpu::SI::SPEED_OF_LIGHT_SI),
```

Comment:
I am a bit suprised the `::picongpu::` prefix is necessary - it should be possible to skip that in this scope.